docs/developers/json-rpc/cluster.md

```diff
@@ -0,0 +1,151 @@
+# Cluster
+
+Maybe you need a lot of evmos.  Fire yourself up a large server, the more iops the better.  Adjust replicas for taste, and note that you'll want to restart in a rolling way that leaves 10m between restarts, about every 24 hours for ideal performance.  This will not give you archive data.
```

Comment:
this is great, I think it needs more context since we don't have any kubernetes in our docs

I also think that this should probably go next to full node guides rather than the RPC section

I think we have to rename this to "node operators" rather than just validators
<img width="156" alt="image" src="https://user-images.githubusercontent.com/8366960/167423832-8bc030a4-c3a5-4ab6-a6b2-098ede4e1678.png">


If you want to go as far as making a whole guide for https://github.com/notional-labs/cosmosia on Evmos @faddat in our docs I would welcome it

Reply:
wow, I'd be super happy to!